custom/userAuth.js

```diff
@@ -14,6 +14,8 @@ const log = require('../server/logger')
 const router = require('express-promise-router')()
 const domains = new Set(process.env.APPROVED_DOMAINS.split(/,\s?/g))
```

Comment:
cleanup nit: can remove this line since the variable's now unused. can also remove the reference to `APPROVED_DOMAINS` in our section of the README.

Reply:
Done, on both counts.